includes/functions.php

```diff
@@ -580,7 +580,10 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         if (host_exists($host) === false) {
             $device_id = dbInsert($device, 'devices');
             if ($device_id) {
-                return($device_id);
+                if ($config['oxidized']['enabled'] === TRUE && $config['oxidized']['reload_nodes'] === TRUE && isset($config['oxidized']['url'])) {
```

Comment:
Worth moving this if check line into the function and just call the function. That way if we reuse the function we don't need to duplicate the if check line.